man/man8/zpool.8

```diff
@@ -1760,12 +1763,15 @@ pool, in addition to the pool-wide statistics.
 Omit statistics since boot.
 Normally the first line of output reports the statistics since boot.
 This option suppresses that first line of output.
+.It Fl n
+Print headers only once when passed 
+.Ar interval
 .It Fl w
 Display latency histograms:
 .Pp
 .Ar total_wait :
 Total IO time (queuing + disk IO time).
-.Ar disk_wait :
+.Ar disk_wait 
```

Comment:
Typo?

Reply:
@gmelikov Could you please review my changes after your code review?